contrib/babelfishpg_tsql/src/dbcmds.c

```diff
@@ -368,7 +369,7 @@ do_create_bbf_db(const char *dbname, List *options, const char *owner)
 	int16		dbid;
```

Comment:
Let's get rid of multiple downcase_identifier() calls, since case sensitive db_name is only required for only one instance.